### ISSUE TITLE
I fail to reproduce the standard Crossentropy classification accuracy

### ISSUE THREAD

**Issue by sndnyang** [lhao499/HDGE#3]
Hi
The idea is really interesting! and shows good results.

I just had a question,  I can't reproduce the  baseline  Supervised learning accuracy (not JEM)
CIFAR10
95.8
CIFAR100
79.9

using the following (disable the contrast to only use  cross entropy)

CUDA_VISIBLE_DEVICES=2 python train_wrn.py --dataset cifar100 --pyxce 1.0 --pxycontrast .0 --width 10 --depth 28 --warmup_iters -1 --lr .1 --norm batch --optimizer sgd --batch_size 128 --dropout_rate .3 --weight_decay .0005 --log_dir ./save --id note

I just got  valid accuracy  77.1,  test accuracy  75  on cifar100.  I use  n_valid=1000  to use more training data. But it still has a gap.

**Comment by sndnyang** [comment]
on cifar10, I also got a bit lower result, 95   <  95.8.  

Do you have any suggestions?

Thank you in advance.

**Comment by sndnyang** [comment]
OK, I got 79.0% now.
